packages/mwp-api-proxy-plugin/src/handler.js

```diff
@@ -27,9 +27,12 @@ export function parseRequestQueries(request) {
  * The default API proxy handler - just calls the decorated `proxyApi$` method
  * and serializes the API responses
  */
-export default (request, reply) => {
-	request.proxyApi$(parseRequestQueries(request)).subscribe(
-		responses => reply(JSON.stringify({ responses })).type('application/json'),
-		err => reply(err) // 500 error - will only be thrown on bad implementation
+export default (request, h) => {
+	const handleResponses = responses =>
+		h.response({ responses }).type('application/json');
+
+	return request.proxyApi$(parseRequestQueries(request)).toPromise().then(
+		handleResponses,
+		err => err // 500 error - will only be thrown on bad implementation
```

Comment:
actually, thinking about this a bit more, I don't think we need this line, since the error will be passed along anyway. However, it's not harmful and I don't want to open any cans of worms, so don't worry about it